### DISCUSSION TITLE
Hide integrations inside 'block insert' menu

### DISCUSSION THREAD

**Discussion by hazzuk** [outline/outline#7645] [Ideas]
## Problem

Outline has a **huge** number of integrations and this will likely continue to grow. However I'm not going to use 80% of these integrations, but they still take up a huge area inside the 'block insert' menu.

![image](https://github.com/user-attachments/assets/21b9caf3-88a9-43e5-947a-f91c7eaf6c6e)

## Solution

Allow admins to (either in workspace settings or deployment environment variables) hide integrations they don't want to display to users.

Note, this wouldn't disable the integration and a user could still paste a supported integration link to embed the hidden integration. But the 'block insert' menu would become a reasonable size and only display options the user will find useful.

**Comment by sohnemann** [comment]
I would really like this feature as well. 😊
Right now there are so many options which are quite confusing for some users.
